requirements/base.txt

```diff
@@ -41,7 +41,6 @@ suds==0.4
 tinys3==0.1.11
 unipath>=1.1,<=2.0
 vobject==0.9.1
-wagtail==1.8.1
```

Comment:
Oh, weird, nice catch. How did I miss updating this and just duplicate it? Minor request: could you move the `wagtail` requirement up here so as to be alphabetized properly?